mockkube/src/main/java/io/strimzi/test/mockkube/MockBuilder.java

```diff
@@ -298,7 +296,7 @@ protected void nameScopedMocks(String resourceName, R resource) {
         if (Readiness.isReadinessApplicable(resourceTypeClass)) {
             mockIsReady(resourceName, resource);
         }
-        try {
+        /*try {
```

Comment:
Should this be commented out? Should it be either used or deleted instead?

Reply:
I am just testing whether this bricks other tests.